### PR TITLE
New package: languagetool-4.8

### PR DESCRIPTION
**Pull request by Eluminae**

### CODE REVIEW

File: srcpkgs/languagetool/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+depends="virtual?java-environment"
+short_desc="Checks your writing in more than 20 languages"
+maintainer="Stacy Harper <contact@stacyhaper.net>"
+license="LGPL"
+homepage="https://www.languagetool.org/"
+distfiles="https://www.languagetool.org/download/LanguageTool-${version}.zip"
+checksum=b4775e88fd90d6389f86023600c4c0e16a9cc00cfe8f093bf1f0c2bbf2a1904d
+
+do_install() {
+	cd "$wrksrc"
```
Comment (Chocimier) [inline srcpkgs/languagetool/template]: Already here.

File: srcpkgs/languagetool/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'languagetool'
+pkgname=languagetool
+version=4.8
+revision=1
+archs=noarch
+wrksrc="LanguageTool-${version}"
+makedepends="unzip rsync"
```
[the remark this answers is not in the document]
Comment (Eluminae) [reply srcpkgs/languagetool/template]:
```
=> ERROR: LanguageTool-4.8_1: cannot find unzip bin for extraction.
```
Are you sure ?

File: srcpkgs/languagetool/template [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+	vmkdir /usr/share/java/$pkgname
+	vcopy *.jar /usr/share/java/$pkgname;
+	vcopy libs/*.jar /usr/share/java/$pkgname;
+	rm -rf *.jar libs
+	vcopy "$wrksrc" /usr/share/$pkgname/
+	vbin "$FILESDIR"/languagetool
+}
+
+post_install() {
+	vlicense COPYING.txt
```
[the remark this answers is not in the document]
Comment (Eluminae) [reply srcpkgs/languagetool/template]: ok, i removed it.

File: srcpkgs/languagetool/template (hunk 1 above)
Comment (Eluminae) [reply srcpkgs/languagetool/template]: nice catch

File: srcpkgs/languagetool/files/languagetool [hunk 4]
```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```
[the remark this answers is not in the document]
Comment (Eluminae) [reply srcpkgs/languagetool/files/languagetool]: It came from aur, I'll to fetch it !

File: srcpkgs/languagetool/template [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'languagetool'
+pkgname=languagetool
```
[the remark this answers is not in the document]
Comment (Eluminae) [reply srcpkgs/languagetool/template]: Oh okay ! Didn't knew

File: srcpkgs/languagetool/files/languagetool (hunk 4 above)
[the remark this answers is not in the document]
Comment (Eluminae) [reply srcpkgs/languagetool/files/languagetool]: I dunno how to download a zip or tarball of https://git.archlinux.org/svntogit/community.git/tree/trunk?h=packages/languagetool
I used a curl command on the do_install as in libglvnd packages but this seems not an elegant way.

File: srcpkgs/languagetool/template (hunk 2 above)
[the remark this answers is not in the document]
Comment (Eluminae) [reply srcpkgs/languagetool/template]: I removed rsync, my bad

File: srcpkgs/languagetool/files/languagetool (hunk 4 above)
[the remark this answers is not in the document]
Comment (Chocimier) [reply srcpkgs/languagetool/files/languagetool]: Something similar to tty-clock's license will do.

File: srcpkgs/languagetool/template (hunk 2 above)
[the remark this answers is not in the document]
Comment (Chocimier) [reply srcpkgs/languagetool/template]: Oh, right, it is needed indeed.

File: srcpkgs/LanguageTool/template [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'LanguageTool'
+pkgname=LanguageTool
+version=4.8
+revision=1
+archs=noarch
+wrksrc="LanguageTool-${version}"
```
Comment (Chocimier) [inline srcpkgs/LanguageTool/template]: This is default now.

File: srcpkgs/languagetool/files/languagetool (hunk 4 above)
[the remark this answers is not in the document]
Comment (Eluminae) [reply srcpkgs/languagetool/files/languagetool]: Pretty nice ! The "skip_extraction" was the trick :)

File: srcpkgs/LanguageTool/template (hunk 6 above)
Comment (Eluminae) [reply srcpkgs/LanguageTool/template]: totally true! Changed

**Comment by Chocimier** [comment]
Either installation path or starting script need to be adjusted to other one to make it work.

**Comment by Eluminae** [comment]
Shit. I guess the renaming from "languagetool" to "LanguageTool" breaked the folder structure. I'm on it

**Comment by Eluminae** [comment]
This was it. It is now fixed.

**Pull request merged**